src/context.ts

```diff
@@ -10,7 +10,7 @@ import {
 import {
     type Chat,
     type ChatPermissions,
-    type InlineQueryResult,
+    InlineQueryResult,
```

Comment:
removing the type keyword should also be reverted.

```suggestion
    type InlineQueryResult,
```